lib/liquid/expression.rb

```diff
@@ -8,6 +8,8 @@ class Expression
       'empty'.freeze => :empty?
     }
 
+    INVERTED_LITERALS = LITERALS.invert
```

Comment:
How do we want `nil` inverted, since there are 4 literals that map to it: `nil => nil, 'nil'.freeze => nil, 'null'.freeze => nil, ''.freeze => nil`